documentation/source/user_section/tutorials/spinalcord-segmentation/sct_propseg.rst

```diff
@@ -0,0 +1,29 @@
+Algorithm #1: ``sct_propseg``
+#############################
+
+SCT provides two command-line scripts for segmenting the spinal cord. The first of these is called ``sct_propseg``. This tutorial will explain the how the script works from a high-level theoretical perspective, and then it will provide two usage examples.
+
+Theory
+------
+
+.. figure:: https://raw.githubusercontent.com/spinalcordtoolbox/doc-figures/master/spinalcord_segmentation/optic_steps.png
+   :align: right
+   :figwidth: 20%
+
+   Centerline detection using OptiC
+
+.. figure:: https://raw.githubusercontent.com/spinalcordtoolbox/doc-figures/master/spinalcord_segmentation/mesh_propagation.png
+   :align: right
+   :figwidth: 20%
+
+   3D mesh propagation using PropSeg
```

Comment:
for some reasons they appear inverted (ie: propseg, and then optic), and it should be the other way around:

![image](https://user-images.githubusercontent.com/2482071/120730933-8fc0d680-c4b0-11eb-81c4-68e4d71932ed.png)